lib/middleware/request_tracker.rb

```diff
@@ -166,8 +166,8 @@ def call(env)
 
     request = Rack::Request.new(env)
 
-    if rate_limit(request)
-      result = [429, {}, ["Slow down, too many requests from this IP address"]]
+    if available_in = rate_limit(request)
+      result = [429, { "Retry-After" => available_in }, ["Slow down, too many requests from this IP address"]]
       return result
```

Comment:
Why not combine these two lines?

```suggestion
      return [429, { "Retry-After" => available_in }, ["Slow down, too many requests from this IP address"]]
```